com.mimacom.ddd.pub.pub/plugin.properties

```diff
@@ -1,4 +1,4 @@
 #
 
-pluginName = com.mimacom.ddd.pub.pub
+pluginName = Publication
```

Comment:
Somehow this change did not make it into the product:

<img width="706" alt="image" src="https://user-images.githubusercontent.com/1107810/79302937-f02c3780-7eed-11ea-9717-cca28bb57573.png">

I have trouble understanding how the plugin name is set using the properties file, it does not always seem to work using the `%` reference from the manifest file.